index.html

```diff
@@ -68,9 +68,9 @@ <h3>Tonic</h3>
           </div>
             <ul class="ul">
               <li><span class="canopy">CANOPY</span></li>
-              <img src="images/Counter (1).png" alt="">
+              <img src="images/Counter (1).png" alt="counter image">
```

Comment:
- [ ] There's something to take in mind here: 📜 If the image is not important for understanding the content — for example, it is just decoration or "eye candy" — it should have null alt (alt=""). One way to help determine if an image should have a null alt is to ask yourself: If the image was removed, would the user still get all the information from the page? Please, consider changing this across your project! 🤓 🛠️